client/my-sites/exporter/export-card/index.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import flowRight from 'lodash/flowRight';
```

Comment:
we're now importing directly from `lodash` due to a babel transform that pulls out the modular components for us, so you can now comfortably do this…

```js
import {
	flowRight
} from 'lodash';
```